hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/ClientStatisticsCodec.java

```diff
@@ -36,7 +36,7 @@
 /**
  * The statistics is composed of three parameters.
  * 
- * The first paramteter is the timestamp taken when the statistics collected.
+ * The first parameter is the timestamp taken when the statistics collected.
```

Comment:
These files ending with `Codec` are autogenerated, and we don't allow them to be changed by hand. To change these files, you would have to use this project https://github.com/hazelcast/hazelcast-client-protocol/.

The documentation is located in each codecs definition and can be found here https://github.com/hazelcast/hazelcast-client-protocol/tree/master/protocol-definitions

After correcting the spelling there, you would have to re-generate codecs. Assuming you set up the project and install dependencies for it, the command would be
```
./generator.py -r /path/to/hazelcast
```

If you want, I can do that on behalf of you and push the changes

Reply:
I've added it to https://github.com/stars/jsoref/lists/to-check-spelling, but there are 200 items in it, so I can't guarantee when I'll get to that repository.

If I remember, I'll try to add `Codec\.java$` to my `excludes.txt` so that I don't check it for this repository going forward.